docs/hugo/content/guide/upgrading.md

```diff
@@ -7,6 +7,14 @@ weight: -3
 
 Ensure that you have carefully reviewed the upgrade instructions included in the [release notes](https://github.com/Azure/azure-service-operator/releases) for the release you are upgrading to. If you are upgrading multiple versions, check the release notes for each version. 
 
+### Caution
+
+If upgrading to **v2.0.0**, you *must* be starting from **v2.0.0-beta.5** because we've changed the way CRDs are managed. See [v2.0.0 Breaking Changes](../breaking-changes/breaking-changes-v2.0.0.md) for more information.
```

Comment:
What do you think about something even shorter than this and slightly more structured?

### Caution

**If upgrading to `v2.0.0`:** Carefully review [v2.0.0 Breaking Changes](../breaking-changes/breaking-changes-v2.0.0.md), especially the section on [running `asoctl clean crds`](../breaking-changes/breaking-changes-v2.0.0.md#alpha-crd-versions-have-been-removed)

**If upgrading to `v2.0.0-beta.4`:** Carefully review [v2.0.0-beta.4 Breaking Changes](../breaking-changes/breaking-changes-v2.0.0-beta.4.md), you may need to make some minor modifications to your resources.

Reply:
Yeah, that'd work. I'll change.

Reply:
Done.